### PR TITLE
dmenu: update to 4.9 and drop 32-bit

### PR DESCRIPTION
**Pull request by AndWac**

### CODE REVIEW

File: components/desktop/dmenu/patches/01-config-mk.patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
++PREFIX = /usr
+ MANPREFIX = $(PREFIX)/share/man
+ 
+ X11INC = /usr/X11R6/include
+@@ -28,4 +28,4 @@
+ LDFLAGS  = $(LIBS)
+ 
+ # compiler and linker
+-CC = cc
++CC = gcc
```
Comment (pyhalov) [inline components/desktop/dmenu/patches/01-config-mk.patch]: Should we just remove `CC = cc` or set to `CC ?= gcc` to use oi-userland gcc, not any?

**Review by pyhalov — CHANGES_REQUESTED** [review]

**Comment by AndWac** [comment]
Ok, I removed it.

**Comment by AndWac** [comment]
Are you fine with the change?

**Comment by pyhalov** [comment]
Seems to be fine

**Review by pyhalov — APPROVED** [review]

**Pull request merged**